src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -6,4 +6,14 @@
 
 public class Storage {
     public static final List<User> people = new ArrayList<>();
+
+    public static boolean checkLoginExist(String userLogin) {
+        for (int i = 0; i < Storage.people.size(); i++) {
+            if (Storage.people.get(i).getLogin().equals(userLogin)) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
delete this method

Reply:
Is this method's implementation wrong or I'm just not allowed to add it here? I created it after reading comments on previous review. 
![image](https://github.com/mate-academy/jv-registration-validation/assets/124635285/f41f44d5-fcb9-4597-8bbc-0709c4b9791c)